ext/net/01_net.js

```diff
@@ -86,7 +101,7 @@
 
     get readable() {
       if (this.#readable === undefined) {
-        this.#readable = readableStreamForRid(this.rid);
+        this.#readable = readableStreamForRidUnrefable(this.rid);
```

Comment:
Right now the stream won't get unrefed if you access `.readable` after calling `.unref`. You should unref on creation based on if #unref is set.

Please also add a test for this case.